BLSQ/iaso
# WC2-523: [WEB] NFC card field shows 0 for all beneficiaries even when card is assigned

Explain what problem this PR is resolving
- [WEB] NFC card field shows 0 for all beneficiaries even when card is assigned
Related JIRA tickets : [WC2-523](https://bluesquare.atlassian.net/browse/WC2-523)

## Self proofreading checklist


## Changes
- Add nfc_cards count in the entity dict and serializer
- Link the StorageLogEntry to the right entity if the import is from mobile(entity_uuid)

## How to test
- Create an instance linked to an entity
- Create a StorageDevice linked to that entity with NFC type
- Create one or multiple StorageLogEntry linked the entity, the instance and the created device
- Check the nfc cards on the beneficiary details page and on the instance details page

## Print Screen/video
[Screencast from 2024-10-23 11-18-46.webm](https://github.com/user-attachments/assets/0e4f4e57-a314-4fdd-b97c-5c6d0d571793)


[WC2-523]: https://bluesquare.atlassian.net/browse/WC2-523?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
